### ISSUE TITLE
Handling account activation

### ISSUE THREAD

**Issue by btry** [flyve-mdm/flyve-mdm-web-ui#11]
### Description

The backend has now a demo mode. When enabled, a user who self creates an account will receive an email containing an activation link.

The web interface is the target of the link in the email. Here is an example of hyperlink: 

```
https://frontend.example.com?id=27&validate=6ecffb2b718868e24f6959e10557f82bfcb95b25b9025b338c29fbc61f8827f6
```

id is the ID of an AccountValidation item. validate is the validation pass.

I need the frontend gets these 2 arguments authenticate with tyhe service account and sends :

PUT api/PluginStorkmdmAccountvalidation/27
{
  "input": {
    "_validate": "3157d94b290775deea81ceb32ca99b0246fcf22e89521240fce6c1f225034bc6"
  }
}

Note the underscore in **_validate**. 

If the request succeeds, the acount will lose inactive registered profile (no right) and gain registered profile, with all required rights.

A inactive user is still able to login (needed for some use cases) but will not be able to do anything.

### Expected behaviour
Implemented :) 

### Actual behaviour
Not implemented

**Comment by ajsb85** [comment]
![validate_account](https://cloud.githubusercontent.com/assets/663460/22156141/490aaf44-df32-11e6-9bae-696e48aad573.gif)

**Comment by ajsb85** [comment]
Fixed in 2f49dcbbab04a2dfbfa70365598c261cd045450a
